### ISSUE TITLE
Consider bumping typebox to 0.32.X

### ISSUE THREAD

**Issue by colatkinson** [eropple/fastify-openapi3#5]
The change to do so is pretty simple (eliding the `package.json` changes for brevity):

```diff
diff --git a/src/spec-transforms/find.ts b/src/spec-transforms/find.ts
index 9b7741d..88de653 100644
--- a/src/spec-transforms/find.ts
+++ b/src/spec-transforms/find.ts
@@ -41,7 +41,7 @@ function findTaggedSchemasInSchemas(
     s.oneOf ?? [],
     Object.values(s.properties ?? {}),
     s.additionalProperties,
-    TypeGuard.TArray(s) && [s.items],
+    TypeGuard.IsArray(s) && [s.items],
   ]
     .flat()
     .filter(isNotPrimitive)
```

Of course updating a core peer dep like that will presumably have effects downstream, so I figured I'd check in before submitting a PR or anything.

Please let me know what you think, and if you don't foresee any issues, I can submit a PR.

**Comment by eropple** [comment]
Hey - yeah, this is starting to become a struggle because the upstream library keeps changing things. It's not good. I like Typebox but I'm getting increasingly irked at their unwillingness to not break the API.

I'm doing a renovation now to upgrade, but I'd hope that @sinclairzx81 (who's making a fantastic thing!) might consider some API consistency, because otherwise everything downstream (my own upgrade right now is prompted by Superforms v2 coming into our stack) has to pick a version and nobody's going to pick the same one.

**Comment by eropple** [comment]
Closed in `v0.5.0`.
